### PR TITLE
Add more detail for farm group api calls

### PR DESCRIPTION
**Pull request by thomasgwatson**

### CODE REVIEW

File: APIs.md [hunk 1]
```diff
@@ -65,21 +65,42 @@ Example GraphQL mutation:
...
     "data": {
-      "accessibility": 1,
+      "accessibility": 0, // 0 => closed (invite only), 1 => restrictied (join request requires approval), 2 => open (anyone can instantly join)
+      "description": "This is a long-form description of the group",
       "name": "Test Group",
       "slug": "unique-url-slug",
-      "parentIds": [],
-      "visibility": 1,
+      "parentIds": [], // group-ids for any parent group of this group
+      "visibility": 2, // 0 => hidden (Only members can see), 1 => protected (only members and members of networked groups can see), 2 => public (anyone can see, including external public)
```
Comment (tibetsprague) [inline APIs.md:73]: maybe set the default here to 1

File: APIs.md [hunk 2]
```diff
@@ -65,21 +65,42 @@ Example GraphQL mutation:
   "query": "mutation ($data: GroupInput, $asUserId: ID) { createGroup(data: $data, asUserId: $asUserId) { id name slug } }",
   "variables": {
     "data": {
-      "accessibility": 1,
+      "accessibility": 0, // 0 => closed (invite only), 1 => restrictied (join request requires approval), 2 => open (anyone can instantly join)
```
Comment (tibetsprague) [inline APIs.md:68]: maybe set default here to 1 to show what it is

File: APIs.md [hunk 3]
```diff
@@ -65,21 +65,42 @@ Example GraphQL mutation:
...
+                    opening_hours: "1-5 M-F, 10-6 Weekends", // String descriptor
+                    open_to_public: true, // Boolean,
+                    public_offerings: ["Farmstand", "Gift shop", "Farm tours", "Workshops"]
+                  }
+                }
               }
           }
-      ]
+      ],
+      "moderatorDescriptor": "Steward", //
```
Comment (tibetsprague) [inline APIs.md:94]: add a comment here, and say the default is Moderator

File: APIs.md [hunk 4]
```diff
@@ -65,21 +65,42 @@ Example GraphQL mutation:
...
+      "moderatorDescriptorPlural": "Stewards",
+      "settings": {
+        locationDisplayPrecision: precise, //   precise => precise location displayed, near => location text shows nearby town/city and coordinate shifted, region => location not shown on map at all and location text shows nearby city/region
```
Comment (tibetsprague) [inline APIs.md:97]: we could put all the group settings here, but this is a good next step, dont need every field yet

File: APIs.md [hunk 5]
```diff
@@ -65,21 +65,42 @@ Example GraphQL mutation:
...
+        publicMemberDirectory: false, // Boolean
+      },
+      "typeDescriptor": "Ranch",
```
Comment (tibetsprague) [inline APIs.md:100]: should mention the "Group" is the default

**Review by tibetsprague — APPROVED** [review]
with some optional comments/tweaks

**Review by lorenjohnson — APPROVED** [review]
LGTM👍🏼

**Pull request merged**